src/mailadm/db.py

```diff
@@ -106,6 +106,7 @@ def ensure_tables(self):
                     date INTEGER,
                     ttl INTEGER,
                     token_name TEXT NOT NULL,
+                    warned INTEGER default 0,
```

Comment:
This doesn't seem enough to add that column to the database, we need a small migration. Right now the code throws this error:

```
2022-10-01T15:21:09.414392385Z [2022-10-01 15:21:09 +0000] [1] [INFO] Starting gunicorn 20.1.0
2022-10-01T15:21:09.798155287Z [2022-10-01 15:21:09 +0000] [1] [INFO] Listening at: http://0.0.0.0:3691 (1)
2022-10-01T15:21:09.798421820Z [2022-10-01 15:21:09 +0000] [1] [INFO] Using worker: sync
2022-10-01T15:21:09.811537411Z Exception in thread prune:
2022-10-01T15:21:09.812127949Z Traceback (most recent call last):
2022-10-01T15:21:09.812456207Z   File "/usr/lib/python3.9/threading.py", line 980, in _bootstrap_inner
2022-10-01T15:21:09.813745308Z     self.run()
2022-10-01T15:21:09.813777221Z   File "/usr/lib/python3.9/threading.py", line 917, in run
2022-10-01T15:21:09.825092017Z     self._target(*self._args, **self._kwargs)
2022-10-01T15:21:09.829596380Z   File "/usr/lib/python3.9/site-packages/mailadm/app.py", line 36, in prune_loop
2022-10-01T15:21:09.831456060Z if you want to talk to mailadm with Delta Chat, please run: mailadm setup-bot
2022-10-01T15:21:09.832516727Z     for logmsg in prune(db).get("message"):
2022-10-01T15:21:09.832556384Z   File "/usr/lib/python3.9/site-packages/mailadm/commands.py", line 73, in prune
2022-10-01T15:21:09.845521445Z     expired_users = conn.get_expired_users(sysdate)
2022-10-01T15:21:09.846240439Z   File "/usr/lib/python3.9/site-packages/mailadm/conn.py", line 231, in get_expired_users
2022-10-01T15:21:09.846753982Z     for args in self._sqlconn.execute(q, (sysdate, )).fetchall():
2022-10-01T15:21:09.854563646Z [2022-10-01 15:21:09 +0000] [22] [INFO] Booting worker with pid: 22
2022-10-01T15:21:09.860507887Z sqlite3.OperationalError: no such column: warned
2022-10-01T15:21:09.860891650Z prune thread died, killing everything now
```